CHANGELOG.md

```diff
@@ -1,4 +1,41 @@
-2.27.0 Release notes (2019-5-10)
+X.Y.Z Release notes
+=============================================================
+NOTE: This release is only compatible with Realm Object Server 3.21.0 or later.
+NOTE: These release notes contain release notes from v2.27.0-rc.2 and v2.27.0-rc.3.
+
+### Enhancements
+* Add an optional parameter to the `SubscriptionOptions`: `inclusions` which is an array of linkingObjects properties. This tells subscriptions to include objects linked through these relationships as well (links and lists are already included by default). ([#2296](https://github.com/realm/realm-js/pull/2296)
+* Added `Realm.Sync.localListenerRealms(regex)` to return the list of local Realms downloaded by the global notifier. ([realm-js-private#521](https://github.com/realm/realm-js-private/issues/521)).
+* Encryption now uses hardware optimized functions, which significantly improves the performance of encrypted Realms. ([realm-core#3241](https://github.com/realm/realm-core/pull/3241))
+* Improved query performance when using `in` queries. ([realm-core#3241](https://github.com/realm/realm-core/pull/3241))
+* Improved query performance when querying integer properties with indexes, e.g. primary key properties. ([realm-core#3272](https://github.com/realm/realm-core/pull/3272))
+* Improved write performance when writing changes to disk. ([realm-core#2927](https://github.com/realm/realm-core/pull/2927))
+
+### Fixed
+* Making a query that compares two integer properties could cause a segmentation fault in the server or x86 node apps. ([realm-core#3253](https://github.com/realm/realm-core/issues/3253))
+* Fix an error in the calculation of the downloadable_bytes value supplied to the progress callback. (See sync version 4.0.0)
+* HTTP requests made by the Sync client now always include a `Host: header`, as required by HTTP/1.1, although its value will be empty if no value is specified by the application. (sync v4.2.0)
+* The server no longer rejects subscriptions based on queries with distinct and/or limit clauses. (sync 4.2.0)
```

Comment:
Is this relevant to the realm-js or just the server?

Reply:
I suppose so. If a query was rejected earlier, this fix is very visible to the user.

Reply:
But this is related to what the server does and not something fixed in this version of realm-js?

Reply:
The argument for keeping it is that we dump the server version with this JS version, and that server version has a bug fix which lifts what a query can be. As our users might have had to make workarounds before, it tells them that this bug has been fixed (and to them it is not relevant that it was the server and not the client).

Reply:
This would be the first time we do that then. And it's not clear that it's related to the Server. Other server fixes are not mentioned.
It doesn't seem coherent with what we are doing anywhere else or what we have been doing?

Reply:
Let me remove it.